test/app_web/controllers/issue_controller_test.exs

```diff
@@ -1,16 +1,25 @@
 defmodule AppWeb.IssueControllerTest do
   use AppWeb.ConnCase
-  alias App.{Issue, Repo}
+  alias App.{Issue, User, Repo}
 
   describe "loads issue page" do
     setup do
+      user = %{
+        login: "user_login",
+        user_id: 1,
+        avatar_url: "/avatar.jpg",
+        html_url: "/user/1"
+      }
+      user_changeset = User.changeset(%User{}, user)
+      user = Repo.insert!(user_changeset)
```

Comment:
Maybe pop this logic into an `insert_user` function in a `test/support/test_helpers.ex` file.

Reply:
ref: https://github.com/dwyl/github-backup/issues/102